src/Uno.UWP/UI/Core/Internal/KeyboardStateTracker.cs

```diff
@@ -97,13 +97,5 @@ private static void SetStateOnNonSideKeys(VirtualKey key)
 		}
 	}
 
-	internal static void Reset()
-	{
-		// Clearing state when debugger is attached would
-		// make it hard to debug key events.
-		if (!System.Diagnostics.Debugger.IsAttached)
-		{
-			_keyStates.Clear();
-		}
-	}
+	internal static void Reset() => _keyStates.Clear();
```

Comment:
Is it failing otherwise? If so, @MartinZikmund may have some input for debugging.

Reply:
Yes, this one was failing (the name of the failing test is in the commit message). Like I said above, I would definitely be in favour of these debug-friendly options being explicitly activated (like FeatureConfiguration, but for internal use). The only part I don't like is that they're on by default.

While we're at it, whenever I'm debugging something popup-related,  I tend to manually delete the code that dismisses all popups when the Uno window is alt-tabbed. We could also add this to the list of things that are adjusted when in the "debug-friendly mode".